src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -416,6 +419,8 @@ bool emberAfOperationalCredentialsClusterRemoveFabricCallback(app::CommandHandle
         return true;
     }
 
+    commandObj->FlushAcksRightAwayOnSlowCommand();
```

Comment:
Is fabric removal actually slow, or we calling this for some other reason?  If it's for another reason, I might suggest a different name for the `FlushAcksRightAwayOnSlowCommand` method.

Reply:
It is slow on platforms where PersistentStorageDelegate writes and deletes are slow since removing a fabric causes a large number of storage entries to be mutated, and not always in the simplest/cheapest order